db/migrate/20170812152912_add_unique_constraint_to_craftsmen_employment_id.rb

```diff
@@ -0,0 +1,18 @@
+class AddUniqueConstraintToCraftsmenEmploymentId < ActiveRecord::Migration
+  def change
```

Comment:
That's interesting, because the migrations we wrote on Wednesday used `up` and `down`